test/units/module_utils/aws/test_rds.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# (c) 2017, Michael De La Rue
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.aws.rds import get_rds_db_instance, rds_instance_to_facts, get_rds_snapshot, rds_snap_to_facts, rds_facts_diff
+import unittest
+import datetime
+import copy
+from dateutil.tz import tzutc
+
+
+class FakeResource():
```

Comment:
These tests now run on python 2.7 (though fail on python3 still). I think if we're moving toward using placebo rather than just mocking, it may make more sense to do that here, especially since integration tests won't be run with CI (and running them manually may not happen terribly frequently). Shippable will notice if the placebo recordings lose compatibility with changes made in a PR.